### DISCUSSION TITLE
Which version of ASE needed?

### DISCUSSION THREAD

**Discussion by thienbinh92** [CederGroupHub/chgnet#157] [Q&A]
Hi, I installed ASE version 3.22.1. This error is shown when I try to use StructOptimizer:
```
from chgnet.model import StructOptimizer
opt_sub = relaxer.relax(substrate)

  File ~/miniconda3/lib/python3.9/site-packages/chgnet/model/dynamics.py:257 in relax
    import ase.filters as filter_classes

ModuleNotFoundError: No module named 'ase.filters'
```
Which version of ASE do I need to install?

**Comment by chtchelkatchev** [comment]
development version
pip install --upgrade git+https://gitlab.com/ase/ase.git@master

**Comment by janosh** [comment]
i heard there might be a new pre-release version of ASE coming out today

edit: [and it's out](https://pypi.org/project/ase/3.23.0)! 🎉

**Reply by janosh** [reply]
the manual installation step needed to install `ase` from `main` will no longer be necessary after https://github.com/CederGroupHub/chgnet/pull/159 is merged
